docs/features/authentication.md

```diff
@@ -148,7 +148,7 @@ dns.overwrite.homeserver.client:
     value: 'http://localhost:8008'
 ```
 `name` must be the hostname of the URL that clients use when connecting to the Homeserver.
-In case the hostname is the same as your Matrix domain, you can use `${matrix.domain}` to auto-populate the `value`
+In case the hostname is the same as your Matrix domain, you can use `${server.name}` to auto-populate the `value`
```

Comment:
`${server.name}` does not given you the value of `matrix.domain`, but `server.name`. I think you meant to do something else here.

Reply:
No, this is exactly what we have in our config and which works.

We have the matrix domain `example.com`, but the server is hosted on `matrix.example.com`. When the login request comes to mxisd it has to be forwarded to `matrix.example.com`, which is the `server.name`. Using `matrix.domain` mxisd doesn't have anywhere to forward the request to as the override entry would be for `example.com`.

Reply:
Maybe the phrasing of the whole sentence would need to be redone, but the value is the correct one.

Reply:
Indeed, so you don't want the value of `matrix.domain`, you want the value of `server.name`. There is no reason to change the phrase talking about `matrix.domain`, it's just not applicable to you. You'll want to rework the sentence to talk about both placeholders and what they refer to, or one sentence for each placeholder.

Reply:
Well, `server.name` defaults to `matrix.domain` when not set in the config, so I'd argue that it's always the right when you want to auto-fill it, but I'll see about adding it in another phrase.